CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ All notable changes to this project will be documented in this file.
 - Support configuring the scaler reconcile interval ([#61]).
 - Add simple web-based dashboard that shows the current state and query counts of all clusters.
   This makes it easier to debug state transitions of clusters ([#62]).
+- Add a new cluster state `Unhealthy`.
+  This state is entered once the readiness check of a `Ready` cluster fails (the check is implemented in the scaler implementation).
+  The `Unhealthy` state is kept until the scaler marks that Cluster as ready again.
+  `Unhealthy` clusters won't get any new queries, if all queries are unhealthy queries are queued.
+  <br>Note: Use the now configurable scaler reconcile interval to detect cluster changes quickly ([#63]).
```

Comment:
note: I don't quite understand the sentence in line 15. It is either wrong, or worded weirdly.

note: markdownlint complains about the `<br />` tag. I think we should remove it.

Reply:
You are right, there was a typo: a9726af

Reply:
TIL a backslash also works: 5e5b022